packages/widget-playground/src/defaultWidgetConfig.ts

```diff
@@ -206,6 +206,7 @@ export const widgetBaseConfig: WidgetConfig = {
 export const defaultWidgetConfig: Partial<WidgetConfig> = {
   ...widgetBaseConfig,
   appearance: 'auto',
+  explorerUrl: 'https://scan.li.fi/tx/',
```

Comment:
using a string here to preset the url prefix for the explorer. We could potentially use a function in place here if urls were to also feature a suffix. eg

```
exploreUrl: (tx: string) => `http://prefix/${tx}/suffix`,
```
But for the use case for l LiFi and Jumper scan I don't think that is needed

Reply:
@DNR500 I guess we can provide just the basic url in options like https://scan.li.fi and the rest should follow conventions (`/tx/{hash}`).

Reply:
Makes sense